nix/default.nix

```diff
@@ -1,7 +1,11 @@
 # This is the main entrypoint for the magic-folder derivation.
 { pkgs ? import <nixpkgs> { } }:
 let
-  pkgs' = pkgs.extend (import ./overlays.nix);
+  tahoe-repo = (import ./tahoe-lafs-repo.nix);
+  pkgs' = pkgs.appendOverlays [
+    (import (tahoe-repo + "/nix/overlays.nix"))
+    (import ./overlays.nix)
+  ];
   callPackage = pkgs'.python27Packages.callPackage;
```

Comment:
It seems like this is beyond just "use cachix".  What's the motivation here?

Reply:
See https://github.com/LeastAuthority/magic-folder/pull/432/commits/f60d4a254c08adaefaff278bc234f20bf3b8ac0f

Reply:
I was running into trouble using a new version of attrs (for a separate pull request), because the tahoe dependencies were not picking up changes from the overlay added in this repo.

Reply:
Okay.  Please do file tickets describing the scope/goals of PRs (pre-review) in the future.  It saves reviewer time.